website/docs/core-concepts/stacks/templates/datasources.mdx

```diff
@@ -189,8 +224,9 @@ settings:
 ```
 </File>
 
-After the above `datasources` are defined, you can use them in Atmos stack manifests like this:
+After the above datasources are defined, you can use them in Atmos stack manifests like this:
```

Comment:
IMO, "datasources" is a typo unless we're referring to the field `datasources`. If it's the general concept of "data sources", it should have a space. In other words `datasources` is not a word.